src/AssetsPage/AssetsTable/AssetsTable.test.jsx

```diff
@@ -5,11 +5,15 @@ import { AssetsTable } from './index';
 import { assetActions } from '../../data/constants/actionTypes';
 
 const setAssetToDelete = (asset, wrapper) => {
-  wrapper.setState({ assetToDelete: asset });
+  wrapper.setState({ deletedAsset: asset });
```

Comment:
This just modifies the function to incorporate the change we made with assetToDelete v. deletedAsset in the AssetTable.

Reply:
Nit -- can we update the const name to `setDeletedAsset`?